test-requirements.txt

```diff
@@ -1,3 +1,3 @@
-ansible==2.4.3.0
-molecule==2.10.1
-docker-py==1.10.6
+ansible==2.7.9.0
```

Comment:
Readme still refers to Ansible 2.4.3.0 :)

Reply:
Fixed!